code/stable/nopcm/src/cpp/input.txt

```diff
@@ -22,7 +22,5 @@
 1.0e-10
 # relative tolerance
 1.0e-10
-# temperature of the water (degreeC)
-40.0
```

Comment:
I just wanted to explain this, it happened automatically as a result of building the software artifacts.

However, I think it makes sense, this file originally had 14 variables, but checking the [reader file](https://github.com/JacquesCarette/Drasil/blob/ef6297da096fa03dc27609c59db0c219dd97751f/code/stable/nopcm/src/cpp/InputParameters.cpp#L15), it only reads 13, and the related [sampleInput file](https://github.com/JacquesCarette/Drasil/blob/master/code/datafiles/NoPCM/sampleInput.txt) also only carries these leftover 13.

Reply:
Good that you explained, as I would have asked! It's a bit odd still. @smiths do you know why the temperature is not actually used here?